doc/tools/apigen.py

```diff
@@ -207,7 +207,11 @@ def _parse_module_with_import(self, uri):
         classes : list of str
             A list of (public) class names in the module.
         """
-        mod = __import__(uri, fromlist=[uri])
+        try:
+            mod = __import__(uri, fromlist=[uri])
+        except ImportError:
+            # fix ImportError for submodules in Python 3
+            mod = __import__(uri, fromlist=[])
```

Comment:
Hmm - that's odd.  Sckit-image has a version of `apigen.py` - this is what they are doing : https://github.com/scikit-image/scikit-image/blob/master/doc/tools/apigen.py#L203

Would you mind adding a Python 3 version of the doc build and test to `.travis.yml`?